dist/common/scripts/scylla_raid_setup

```diff
@@ -167,19 +167,18 @@ if __name__ == '__main__':
 
     uuidpath = f'/dev/disk/by-uuid/{uuid}'
 
-    after = 'local-fs.target'
+    after = ''
     wants = ''
     if raid and args.raid_level != '0':
-        after += f' {md_service}'
+        after += f'\nAfter={md_service}'
         wants = f'\nWants={md_service}'
     opt_discard = ''
     if args.online_discard:
         opt_discard = ',discard'
     unit_data = f'''
 [Unit]
 Description=Scylla data directory
-Before=scylla-server.service
-After={after}{wants}
+Before=local-fs.target scylla-server.service{after}{wants}
```

Comment:
IIUC, the change here also intends to remove the empty `After=` settings . but it does not hurt, right? how about put something like
```py
after = wants = f'{md_service}'
# ...
unit_data = f'''
Before=scylla-server.service
...
After={after}
Wants={wants}
'''
```
just trade the readability of the systemd for the better readability of the source code generating it.

Reply:
Applied suggested changes on the unit file generation.